### DISCUSSION TITLE
"Unfinished" marking and crowdsourcing

### DISCUSSION THREAD

**Discussion by mikekasprzak** [ludumdare/ludumdare#2020] [General]
There are a variety of features I'd like to see in the same ballpark, but this one might be one of the more higher priority ones.

One thing I neglected to do for Ludum Dare 38 was give users a way to "Publish" their game as `unfinished`, so I've had to spend a bit of time manually editing the data to correctly reflect this. Ideally, the 2nd part should be something I can let community members do for me. Unfortunately this needs to be handled on a case by case basis, so when a bunch of users flag a game as unfinished, it shouldn't necessarily happen right away (at least not initially). 

Communication is important here, if a game gets marked as unfinished by the community, a user should get the opportunity to appeal. Our criteria for an unfinished game are games that the author chose to remove from the voting, or a game lacking any way to download and play them. A game that really does next to nothing is a valid game, but there may be cases where a game is created to abuse the fact that we let people with games vote, and care needs to be taken here.

Unfinished games are games that shouldn't be part of the voting, but still kept around to have a record. This is important because any posts authored under them NEED the parent to exist for them to work properly. In general they will be hidden, but there will be methods for looking them up.

**Comment by local-minimum** [comment]
In part this relates to #963 but I agree, the first things that come to mind is a `report` system and elevating some trusted users to admins that would have tools to do some stuff like:

* Set unfinished game
* Mute/block spamming user
* Hide offensive post
